icons/insomnia/insomnia-wordmarker.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Replace this icon with the version from their official website: https://insomnia.rest/ -> https://insomnia.rest/images/insomnia-logo.svg
![](https://insomnia.rest/images/insomnia-logo.svg)